test/linalg/arnoldi.jl

```diff
@@ -20,6 +20,7 @@ using Test
                 a = areal
                 b = breal
             end
+            a_evs = eigvals(Array(a))
```

Comment:
Perhaps move this to L45 or so, closer to where it is used?

Reply:
I didn't do that because all the setup stuff is here ... do you think it's too confusing?